proposals/mpi-operator-proposal.md

```diff
@@ -78,6 +83,15 @@ spec:
       containers:
       - name: tensorflow-benchmarks
         image: rongou/tensorflow_benchmarks:latest
```

Comment:
Hi, where could we find the source code of the image?

Reply:
Source code forthcoming, which would include the Dockerfile to build this image.

Reply:
Thanks :clap: